src/App.tsx

```diff
@@ -15,71 +15,88 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component {
```

Comment:
you can here add types for props and state
```suggestion
export class App extends Component<generic> {
```